.travis.yml

```diff
@@ -1,22 +1,18 @@
+dist: xenial
 language: elixir
 cache: apt
 env:
-  - EVM_EMACS=24.3
-  - EVM_EMACS=24.4
-  - EVM_EMACS=24.5
-  - EVM_EMACS=25.1
-  - EVM_EMACS=25.2
-matrix:
-  include:
-    - elixir: 1.0.5
-      otp_release: 17.4
-    - elixir: 1.4.3
-      otp_release: 19.0
-    - elixir: 1.6.0
-      otp_release: 20.1
+  - EVM_EMACS=26.3
```

Comment:
We should keep emacs versions starting at 24.3 in the build, I know it's a pain but ... it's still in use.

Reply:
So I couldn't get 24.5 to run (which is what is broken on master as well). Any idea how to fix?

Reply:
@Trevoke since its broken on master and it is an indentation test, not a faces test I'd be okay to merge this even with the 24.5 test broken. Wdyt?

@jsmestad would you mind reverting the matrix change for now?

Reply:
@victorolinasc I won't have many free cycles for quite awhile to do that, but I am good with the change.

You should have access to modify the branch.